common/utils/utils.go

```diff
@@ -492,3 +494,102 @@ func DialectToTarget(dialect string) string {
 	}
 	return constants.TargetSpanner
 }
+
+const alphabet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890"
+
+func randomString(n int) string {
+	var sb strings.Builder
+	k := len(alphabet)
+
+	for i := 0; i < n; i++ {
+		c := alphabet[rand.Intn(k)]
+		sb.WriteByte(c)
+	}
+
+	return sb.String()
+}
+
+func PopulateMigrationData(conv *internal.Conv, driver, targetDb string) {
+
+	migrationRequestId := "HB" + randomString(14)
+
+	numTables := int32(len(conv.SrcSchema))
+	migrationData := migration.MigrationData{
+		MigrationRequestId: &migrationRequestId,
+		SchemaPatterns: &migration.MigrationData_SchemaPatterns{
+			NumTables: &numTables,
+		},
+	}
+	switch driver {
+	case constants.PGDUMP:
```

Comment:
indent is incorrect for all case statements here and below. Factor out this switch into its own method.